experimental/FTheoryTools/docs/src/hypersurface.md

```diff
@@ -76,7 +76,7 @@ Completeness is an expensive check. Therefore, we provide an optional argument w
  demonstrate this:
 ```@docs
 hypersurface_model(base::NormalToricVariety; completeness_check::Bool = true)
-hypersurface_model(base::NormalToricVariety, fiber_ambient_space::NormalToricVariety, D1::ToricDivisorClass, D2::ToricDivisorClass; completeness_check::Bool = true)
+hypersurface_model(base::NormalToricVariety, fiber_ambient_space::NormalToricVariety, fiber_twist_divisor_classes::Vector{ToricDivisorClass}; completeness_check::Bool = true)
```

Comment:
Is this not what we were planning to call `fiber_twist_matrix`, at least in the JSON files? I imagine this name could match that

Reply:
As discussed in zoom, we are now fine with the name.